src/sentry/utils/committers.py

```diff
@@ -158,23 +166,17 @@ def _get_committers(
             if limit == 0:
                 break
 
-    # organize them by this heuristic (first frame is worth 5 points, second is worth 4, etc.)
-    sorted_committers = sorted(committers.items(), key=operator.itemgetter(1))
-
-    users_by_author: Mapping[str, Author] = get_users_for_commits([c for c, _ in commits])
-
-    user_dicts: Sequence[AuthorCommits] = [
+    author_users: Mapping[str, Author] = get_users_for_commits([c for c, _ in commits])
+    return [
         {
-            "author": users_by_author.get(str(author_id), {}),
+            "author": author_users.get(str(author_id)),
```

Comment:
I'd say we should probably continue returning an empty dict here. Not sure whether this actually happens, but would be better to avoid logic changes in a typing pr.

Reply:
Hmm.. actually, the default behavior was to return `None`:
https://github.com/getsentry/sentry/blob/4b591ecf9703914b98f9e1a9c266b57a2d73cb9c/src/sentry/utils/committers.py#L136

I introduced the returning empty dict which is probably not causing issues (yet). I'll scan through and make sure we go to returning `None` throughout.